init.js

```diff
@@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
     return db.Role.create({ name: 'public' });
   })
   .then(function () {
-    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
+    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
```

Comment:
No it shows /index. I'd have to add a conditional to make that happen.
Shouldn't be too hard, I'll add it in tonight

On Monday, September 28, 2015, Cody B. Daig <notifications@github.com>
wrote:

> In init.js
> <https://github.com/nobsjs/nobsjs/pull/256#discussion_r40601152>:
>
> > @@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
> >      return db.Role.create({ name: 'public' });
> >    })
> >    .then(function () {
> > -    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
> > +    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
>
> with /index the url to the user is still / right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nobsjs/nobsjs/pull/256/files#r40601152>.
>